sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/utils.py

```diff
@@ -377,14 +379,17 @@ def raise_exception(msg, target):
                     message="Please install 'azure-ai-inference' package to use SystemMessage, AssistantMessage"
                 ) from ex
 
-            if isinstance(messages[0], ChatRequestMessage) and not isinstance(
+            if isinstance(message, ChatRequestMessage) and not isinstance(
                 message, (UserMessage, AssistantMessage, SystemMessage)
             ):
                 raise_exception(
                     f"Messages must be a strongly typed class of ChatRequestMessage. Message number: {num}",
                     ErrorTarget.CONTENT_SAFETY_MULTIMODAL_EVALUATOR,
                 )
-
+            if isinstance(message, AssistantMessage):
+                assistantMessageCount = assistantMessageCount + 1
```

Comment:
NIT:
```suggestion
                assistantMessageCount += 1
```
This operation is better because it does `INPLACE_ADD` instead of `BINARY_ADD`